app/views/comments/_comment.html.erb

```diff
@@ -28,7 +29,9 @@
   ">
 
   <div class="voters">
-    <label for="comment_folder_<%= comment.short_id %>" class="comment_folder"></label>
+    <% if !defined?(show_folder_control) || show_folder_control %>
```

Comment:
```suggestion
    <% if show_folder_control %>
```

It shouldn't be possible for it to be undefined because of the default in the "signature".

Reply:
I needed to include a check for `!defined?(show_folder_control)` here because the heinous_inline_partial(comments/_comment.html.erb) in comments/_threads.html.erb already includes strict locals at the top of that view, and so when the heinouse_inline_partial inlines comments/_comment.html.erb along with it's locals, it gets ignored, causing an error to appear for that variable when visiting http://127.0.0.1:3000/threads

Reply:
Ah, I hadn't realized that possible context. Improving the consistency is a great approach.

I don't think there's any fix for the heinous views, they're living down to their name.